packages/unit-testing/README.md

```diff
@@ -1,45 +1,314 @@
 # SuiteCloud Unit Testing
 Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestjs.io/) for your SDF projects.
 
+## Features
+- Provides a default configuration to run unit tests with Jest in SDF projects.
+- Supports unit testing for SuiteScript 2.X files.
+- Provides SuiteScript 2.X stubs.
+>💡 For NetSuite 2020.1, only the **N/record** module is supported.
+- Allows you to create custom stubs for SuiteScript 2.X or custom modules.
+- Supports code coverage capabilities through Jest.
+
 ## Prerequisites
-The following software is required to use SuiteCloud Unit Testing package:
 - Node.js version 12.14.0 LTS or greater
-- [Jest](https://jestjs.io/docs/en/getting-started.html)
-- A `__tests__` folder in the root of your SDF project.
-- A `jest.config.js` file in the root of your SDF project.
+- An SDF project
 
-## `jest.config.js` File Examples
-The `jest.config.js` file must follow a specific structure. Depending on your SDF project type, check one of the following examples:
+## Getting Started
+> ⚠ SuiteCloud Unit Testing is installed as a `devDependency`.
+
+To get started with SuiteCloud Unit Testing, do the following:
 
-- For ACP:
+1. Inside of your SDF project folder, create a `src` folder.
+2. Move your project files inside of the `src` folder.
+3. To initialize NPM, from the root of your SDF project folder, run `npm init`.
+>💡 A `package.json` file is created in your SDF project folder.
+4. In your `package.json` file, paste the following code:
+```json
+{
+  "scripts": {
+    "test": "jest"
+  }
+}
+```
+5. From the root of your SDF project folder, run the followig command:
+```
+npm install --save-dev @oracle/suitecloud-unit-testing jest
+```
+6. Create a `__tests__` folder, inside of the root of your SDF project folder.
+7. Create a `sample-test.js` file, inside of the `__tests__` folder, with the following content:
+```javascript
+describe('Basic jest test with simple assert', () => {
+    it('should assert stings are equal', () => {
+        const a = 'foobar';
+        const b = 'foobar';
+        expect(a).toMatch(b);
+    });
+});
+```
+8. From the root of your SDF project folder, run `npm test`
+to run your test. You should see an output similar to following:
+```
+PASS  __tests__/BasicJest.test.js
+  Basic jest test with simple assert
+    √ should assert stings are equal (2ms)
 ```
+>💡 If you want to run tests with coverage, from the root of your SDF project folder, run `npm test --coverage`.
+
+**You successfully ran your first test for an SDF project!**
+
+➡ If you use SuiteCloud CLI for Node.js, you can install SuiteCloud Unit Testing when running the [`project:create`](../../packages/node-cli/README.md/##Commands) command by following the questions prompted.
+
+## Additional Configuration
+To properly structure and run your test files, create a `jest.config.js` file, inside of the root of your SDF project folder.
```

Comment:
I would change to:
To properly run your tests against SuiteScript 2.X files, create a jest.config.js file, inside of the root of your SDF project folder.